gdown/download.py

```diff
@@ -6,9 +6,10 @@
 import tempfile
 import textwrap
 import time
-import urllib
+import urllib.parse
```

Comment:
Because that is the proper import. Even before, we never used anything from `urllib`, but only from `urllib.parse`. If you look at the [Python documentation](https://docs.python.org/3/library/urllib.html), `urllib` is just the wrapper namespace and points to its submodules.

Reply:
Seems it was working because of the import of requests.